src/less/components/resource.less

```diff
@@ -1,16 +1,95 @@
+.toolbar-secondary-wrapper{
+  padding-bottom: 10px;
+}
+
+.toolbar-resource-secondary{
+  max-width: 1110px;
+  margin: 0 auto;
+  padding: 0 20px;
+
+  i{
+    border-radius: 0px;
+    height: 16px;
+    width: 16px;
+    color: #2a6ebb;
+  }
+
+  a{
+    color: rgb(42, 110, 187);
+    font-family: @font-family-sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    height: 18px;
+    letter-spacing: 0.4px;
+    line-height: 18px;
+    text-transform: uppercase;
+    width: 96px;
+  }
+}
+
+
 .module-resource {
   border: none;
-  margin-bottom: @margin__module;
+  //margin-bottom: @margin__module;
+  //padding-bottom: @margin__module;
 
   // Additional padding for desktop only
   @media (min-width: @screen-sm-min) {
     padding-left: @padding-left__module--resource;
     padding-right: @padding-right__module--resource;
   }
+
+  padding-left: @padding-left__module--resource;
+  padding-right: @padding-right__module--resource;
+
   
   .actions {
     top: 0;
     right: 0;
+
+    ul{
+      float: right;
+    }
+  }
+
+  .module-small-title{
+    height: 24px;
+    color: rgb(95, 104, 109);
+    font-size: 16px;
+    font-family: @font-family-sans-serif; 
+    font-weight: normal;
+    line-height: 24px;
+    padding-top: 10px;
+  }
```

Comment:
I'd recommend to use variables for font's at least (found from `variables/fonts.less`). `@font-size-slightly-smaller-base` and `@line-height-base` should be fine in here. Also I would avoid hard coded height for elements if there's no good reason for that (like horizontal scroll etc.)

Also most of colors has variable in `variables/palette.less` and those should be used (if there isn't suitable one, maybe one should be added).

Reply:
Resolved in [4bd529b](https://github.com/vrk-kpa/opendata-assets/pull/38/commits/4bd529b924677e0b078b615982ed2ec2625bf2c1)